profiles-sdk-examples/main-app/README.md

```diff
@@ -0,0 +1,11 @@
+# Profiles Examples Main Application
+
+For instructions building and running see parent the [README.md](../README.md). This example is structured as CLI
```

Comment:
I have no idea what this sentence is trying to say. Maybe
"For instructions on how to build and run Profile examples see the parent README file. This Profile main application example is structure as a CLI ...